src/odemis/util/test/angleres_test.py

```diff
@@ -90,6 +93,25 @@ def setUp(self):
         white_data_2500.metadata[model.MD_PIXEL_SIZE] = white_pxs_2500
         self.white_data_2500 = white_data_2500
 
+    def test_invertedMirror_AR2polar(self):
+        data_invMirror = self.data_invMir
```

Comment:
If you don't need the dimensions, you can just write `data_invMirror = ensure2DImage(self.data_invMir[0])`

Reply:
I need the dimensions of the image so I didn't change this part.

Reply:
The proposed code replaces the 3 lines. Your code really doesn't look like it needs the Y,X values afterwards, so please double check.

Reply:
in _flipDataIfMirrorFlipped
    data = data[::-1, :]
IndexError: too many indices for array